textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -42,21 +44,57 @@
     private List<Attribute> searchInAttributes;
     private ITuple sourceTuple;
     private Schema returnSchema;
+    private NlpConstant inputNlpConstant = null;
+    private String flag = null;
 
 
-    public static final String NE_NUMBER = "Number";
-    public static final String NE_LOCATION = "Location";
-    public static final String NE_PERSON = "Person";
-    public static final String NE_ORGANIZATION = "Organization";
-    public static final String NE_MONEY = "Money";
-    public static final String NE_PERCENT = "Percent";
-    public static final String NE_DATE = "Date";
-    public static final String NE_TIME = "Time";
+    /**
+     * Named Entity Constants: NE_ALL, Number, Location, Person,
+     * Organization, Money, Percent, Date, Time.
+     * Part Of Speech Constants: Noun, Verb, Adjective, Adverb
+     */
+    public enum NlpConstant {
```

Comment:
Do you think `NlpTokenType` sounds better than `NlpConstant`?  I felt the word "Constant" doesn't give us much info....